machine_usage/forms.py

```diff
@@ -78,15 +78,25 @@ def __init__(self, machine, *args, **kwargs):
 
 #Usage Length usage
 class MachineUsageLength(forms.Form):
-    usage_hours = forms.IntegerField(required=True, min_value=0, widget=forms.NumberInput(attrs={'placeholder': 'Hours'}))
-    usage_minutes = forms.IntegerField(required=True, min_value=0, max_value=60, widget=forms.NumberInput(attrs={'placeholder': 'Minutes'}))
+    usage_hours = forms.IntegerField(required=False, min_value=0, widget=forms.NumberInput(attrs={'placeholder': 'Hours'}))
+    usage_minutes = forms.IntegerField(required=False, min_value=0, max_value=60, widget=forms.NumberInput(attrs={'placeholder': 'Minutes'}))

```

Comment:
It might be simpler to just add `initial=0` [parameter](https://docs.djangoproject.com/en/4.1/ref/forms/fields/#initial) so that the form automatically starts with 0 hours and 0 minutes. That way the field requirement is enforced on both the frontend and backend

Reply:
didn't even know that was an option, thanks